### PR TITLE
[TT-7419] Add create version params to swagger documentation

### PR DESCRIPTION
**Pull request by furkansenharputlu**
https://tyktech.atlassian.net/browse/TT-7419

### CODE REVIEW

**Review by Yasintrm — APPROVED** [review]

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4587/merge
Commit:  
Triggered by: pull_request (@furkansenharputlu)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3788568434)

File: swagger.yml [hunk 1]
```diff
@@ -278,6 +278,31 @@ paths:
       tags:
         - APIs
       operationId: createApi
+      parameters:
```
Comment (jeffy-mathew) [inline swagger.yml:281]: shall we group these params under `#/components/schemas/`?

**Review by jeffy-mathew — APPROVED** [review]

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/heads/master
Commit: c39290ef4e53aa56a3b3899957283c2cae6c638a [TT-7419] Add create version params to swagger documentation (#4587)

https://tyktech.atlassian.net/browse/TT-7419
Triggered by: push (@furkansenharputlu)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3789057726)

**Pull request merged**